odoo/tools/safe_eval.py

```diff
@@ -280,7 +280,7 @@ def _import(name, globals=None, locals=None, fromlist=None, level=-1):
     'zip': zip,
     'Exception': Exception,
 }
-def safe_eval(expr, globals_dict=None, locals_dict=None, mode="eval", nocopy=False, locals_builtins=False):
+def safe_eval(expr, globals_dict=None, locals_dict=None, mode="eval", nocopy=False, locals_builtins=False, filename=None):
```

Comment:
might be a good time to start documenting params in the docstring, as `filename` may not be obvious for all callers :-)

using the RST docstring format: 
```
    :param filename: optional pseudo-filename for the compiled expression,
                     displayed for example in traceback frames
```

Reply:
Or at least update the pseudo-signature (or maybe remove it and replace it by info fields? It's rather wrong as-is).